src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/controller/MaintainSupplierPricingController.kt

```diff
@@ -37,7 +38,7 @@ class MaintainSupplierPricingController(@Autowired val supplierPricingService: S
   data class PriceForm(
     @get: NotNull(message = "Invalid message id")
     val moveId: String,
-    @get: NotNull(message = "Add a price")
+    @get: Pattern(regexp = "^[0-9]{1,4}(\\.[0-9]{0,2})?\$", message = "Invalid rate")
```

Comment:
Prior to this, this would allow 0.001 through as this is greater than zero but it was then getting truncated so in effect allowing a price of zero to get through.